dependency-check-suppressions.xml

```diff
@@ -21,5 +21,14 @@
     <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson-databind@.*$</packageUrl>
     <vulnerabilityName>CVE-2019-12814</vulnerabilityName>
   </suppress>
-  
+
+  <suppress>
+    <notes><![CDATA[
+   relates to form based authentication, not relevant to us, fix will be picked up in next spring boot update
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.apache\.tomcat\.embed.*$</packageUrl>
+    <cve>CVE-2019-17563</cve>
+    <cve>CVE-2019-12418</cve>
```

Comment:
you can use 9.0.24 tomcat version instead of adding suppression https://github.com/hmcts/bulk-scan-processor/blob/master/build.gradle#L191-L193

Reply:
Felt not worth it for plum, unless you insist :)

Reply:
Especially because there is a high chance that it will get forgotten when dependabot upgrades spring boot